python-lib/language_support.py

```diff
@@ -84,3 +84,47 @@
 
 Models with Creative Commons licenses are not included because this plugin is licensed under Apache-2
 """
+
+SPACY_LANGUAGE_MODELS_LEMMATIZATION = ["en", "es", "nb", "fr", "de", "ru", "pl"]
+
+"""list: Languages that have a SpaCy pre-trained model with a Lemmatizer component"""
+
+SPACY_LANGUAGE_MODELS_MORPHOLOGIZER = ["es", "nb", "fr", "de", "ru", "pl"]
+
+"""list: Languages that have a SpaCy pre-trained model with a Morphologizer component"""
```

Comment:
What is the morphologizer for?

Reply:
When using a pre-trained pipeline, you need to activate:
-either morphologizer 
-either tagger+attribute ruler  (just for english in our case)
depending on the pre-trained pipeline built-in components

Reply:
Aha indeed! I added it in 09f5159